src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div
+      class="calendar calendar--start-day-mon calendar--month-length-30"
+    >
```

Comment:
There was suggestion 

```
<div class="
      calendar 
      calendar__start-day--mon 
      calendar__month-length--30
    ">
```
But I guess it was changed by linter, so never mind

Reply:
Bem lint does not pass in this case. I asked a question in the chat and Misha told me that the test has errors and that in order to bypass the test you can specify all classes on one line, just move the closing tag to a new line, as I did.